text_collector_examples/deb_check_package_updates.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+/usr/bin/apt-get update
```

Comment:
I would prefer to not have this script trigger apt-get update.  There are already cron jobs on many systems that deal with updating the package list(s).